tests/acceptance/helpers/ldapHelper.js

```diff
@@ -0,0 +1,187 @@
+const { client } = require('nightwatch-api')
+const ldap = require('ldap')
+const { join } = require('./path')
+const fs = require('fs-extra')
+
+const userHelper = require('./userSettings')
+
+exports.createClient = function (url) {
+  return new Promise((resolve, reject) => {
+    const ldapClient = ldap.createClient({
+      url: url || client.globals.ldap_url // 'ldap://127.0.0.1'
+    })
+
+    ldapClient.bind('cn=admin,dc=owncloud,dc=com', 'admin', err => {
```

Comment:
should be configurable see e.g. https://github.com/owncloud/core/pull/36791/files#diff-d8f5321aed99cdfd86ee188d13edb329R85